modules/blocks.php

```diff
@@ -190,3 +191,70 @@ function jetpack_slideshow_block_load_assets( $attr, $content ) {
 	Jetpack_Gutenberg::load_assets_as_required( 'slideshow', $dependencies );
 	return $content;
 }
+
+/**
+ * Business Hours Block.
+ */
+jetpack_register_block(
+	'business-hours',
+	array( 'render_callback' => 'jetpack_business_hours_render' )
+);
+/**
+ * Business Hours Block dynamic rending of the glock.
+ *
+ * @param array  $attr - Array containing the business hours block attributes.
+ * @param string $content - String containing the business hours block content.
+ *
+ * @return string
+ */
+function jetpack_business_hours_render( $attributes, $content ) {
+	global $wp_locale;
+
+	if ( empty( $attributes['hours'] ) || ! is_array( $attributes['hours'] ) ) {
+		return $content;
+	}
+
+	$start_of_week = (int) get_option( 'start_of_week', 0 );
+	$time_format = get_option( 'time_format' );
+	$today = current_time( 'D' );
+	$content = '<dl class="business-hours built-by-php">';
+
+	$days = array( 'Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat' );
+
+	if ( $start_of_week ) {
+		$chunk1 = array_slice( $attributes['hours'], 0, $start_of_week );
+		$chunk2 = array_slice( $attributes['hours'], $start_of_week );
+		$attributes['hours'] = array_merge( $chunk2, $chunk1 );
+	}
+
+	foreach ( $attributes['hours'] as $day => $hours ) {
+		$opening = strtotime( $hours['opening'] );
+		$closing = strtotime( $hours['closing'] );
+
+		$content .= '<dt class="' . esc_attr( $day ) . '">' . $wp_locale->get_weekday( array_search( $day, $days ) ) . '</dt>';
+		$content .= '<dd class="' . esc_attr( $day ) . '">';
+		if ( $hours['opening'] && $hours['closing'] ) {
+			$content .= date( $time_format, $opening );
```

Comment:
I wonder if this would be better as one translatable string. I can imagine that some languages use em dashes in that situation, but others may use `/` or even words like `from %s to %s` for example.